gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LRGradleCore.java

```diff
@@ -77,16 +78,15 @@ public static LRGradleCore getDefault()
         return plugin;
     }
 
-
     @SuppressWarnings( "unchecked" )
-    public static <T> T getToolingModel( Class<?> modelClass, GradleProject gradleProject )
+    public static <T> T getToolingModel( Class<?> modelClass, IProject gradleProject )
```

Comment:
Is there a way to do this with buildship APIs and we don't have to have our own custom version?